content/en/security/application_security/api-inventory/_index.md

```diff
@@ -94,21 +98,28 @@ Datadog marks an endpoint as public if the client IP address is outside these ra
 
 See [Configuring a client IP header][6] for more information on the required library configuration.
 
-### Unauthenticated endpoint
+### Endpoint Authentication
 
 Authentication is determined by:
 
 - The presence of `Authorization`, `Token` or `X-Api-Key` headers.
 - The presence of a user ID within the trace (for example, the `@usr.id` APM attribute).
 - The request has responded with a 401 or 403 status code.
 
-### Contains exploitable vulnerabilities
+Datadog reports the type of authentication when available in a header through the Authentication Method facet
 
-This risk is determined by [Software Composition Analysis][3] for the service hosting the endpoint.
+#### Supported authentication methods
 
-## Performance impact
+| Category                                          | Category facet   |
+|---------------------------------------------------|------------------|
+| JSON Web Token (JWT)                              | `json_web_token` |
+| Bearer tokens (found in `Authorization` headers)  | `bearer_token`   |
+| Basic Authentication                              | `basic_auth`     |
+| Digest access authentication                      | `digest_auth`    |
+
+### Contains exploitable vulnerabilities
 
-By default, API Security Inventory evaluates every tenth request (10% sample rate).
+This risk is determined by [Code Security][8] for the endpoint and by [Software Composition Analysis][3] for the service it.
```

Comment:
We miss a part in the sentence no? What do you mean by `by [Software Composition Analysis][3] for the service it.`

Reply:
I reworded